src/cpp/fastdds/topic/qos/TopicQos.cpp

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file TopicQos.cpp
+ *
+ */
+
+#include <fastdds/topic/qos/TopicQos.hpp>
+
+using namespace eprosima::fastdds;
+
+TopicQos eprosima::fastdds::TOPIC_QOS_DEFAULT;
+
+TopicQos::TopicQos()
+{
+
+}
+
+TopicQos::~TopicQos()
+{
+
+}
+
+void TopicQos::setQos(const TopicQos& qos, bool first_time)
```

Comment:
Remove these methods if they are not implemented. You can leave a `TODO` comment in the header file